python/cudf/cudf/pandas/_wrappers/pandas.py

```diff
@@ -93,14 +93,16 @@ class _AccessorAttr:
     """
 
     def __init__(self, typ):
-        self.__typ = typ
+        self._typ = typ
+
+    def __set_name__(self, owner, name):
```

Comment:
TIL a new part of the descriptors object model!

https://docs.python.org/3/howto/descriptor.html#closing-thoughts